### PR TITLE
create config namespaces as part of image registry operator bringup

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by adambkaplan** [comment]
good practice to rename the prefixes so the ordering is clear?

**Comment by bparees** [comment]
> good practice to rename the prefixes so the ordering is clear?

the ordering doesn't matter in this case, so renaming everything would be a waste.  and since this is a (hopefully) temporary home, i'm even less concerned.

**Comment by bparees** [comment]
(In general my OCD would agree w/ you that i'd like everything to have an explicit ordering, but in practice we should probably get comfortable w/ having multiple things at the same priority if they do not have a dependency)

File: deploy/01-openshift-config-namespace.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-config-managed
```
Comment (smarterclayton) [inline deploy/01-openshift-config-namespace.yaml]: What is this namespace?

Comment (smarterclayton) [reply deploy/01-openshift-config-namespace.yaml]: I understand the other one, what is this one?

**Pull request merged**